pkg/controllers/nodetemplate/suite_test.go

```diff
@@ -258,7 +258,8 @@ var _ = Describe("AWSNodeTemplateController", func() {
 			ExpectApplied(ctx, env.Client, nodeTemplate)
 			ExpectReconcileSucceeded(ctx, controller, client.ObjectKeyFromObject(nodeTemplate))
 			nodeTemplate = ExpectExists(ctx, env.Client, nodeTemplate)
-			securityGroupsIDs, _ := awsEnv.SecurityGroupProvider.List(ctx, nodeTemplate)
+			securityGroups, _ := awsEnv.SecurityGroupProvider.List(ctx, nodeTemplate)
```

Comment:
We should probably ensure the entire security group struct now, right? Rather than just the ids?